<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addBookButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBookButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterCategoryButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by Category</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeBookButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Book</string>
    </property>
   </widget>
   <widget class="QListWidget" name="bookList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>340</width>
      <height>140</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
